catalog/spatial/registry/registry-admin-modules/registry-admin-remote-ui/src/main/webapp/js/HandlebarsHelpers.js

```diff
@@ -0,0 +1,188 @@
+                    if (!item) {
+                        result = false;
+                    }
+                });
+                if (result) {
+                    return block.fn(this);
+                }
+                else {
+                    return block.inverse(this);
+                }
+            },
+            ifOr: function () {
+                var args = _.flatten(arguments);
+                var items = _.initial(args);
+                var result = false;
+                var block = _.last(args);
+                _.each(items, function (item) {
+                    if (item) {
+                        result = true;
+                    }
+                });
+                if (result) {
+                    return block.fn(this);
+                }
+                else {
+                    return block.inverse(this);
+                }
+            },
+            ifNotAnd: function () {
+                var args = _.flatten(arguments);
+                var items = _.initial(args);
+                var result = true;
+                var block = _.last(args);
+                _.each(items, function (item) {
+                    if (!item) {
+                        result = false;
+                    }
+                });
+                if (result) {
+                    return block.inverse(this);
+                }
+                else {
+                    return block.fn(this);
+                }
+            },
+            ifNotOr: function () {
+                var args = _.flatten(arguments);
+                var items = _.initial(args);
+                var result = false;
+                var block = _.last(args);
+                _.each(items, function (item) {
+                    if (item) {
+                        result = true;
+                    }
+                });
+                if (result) {
+                    return block.inverse(this);
+                }
+                else {
+                    return block.fn(this);
+                }
+            },
+            propertyTitle: function (str) {
+                if (_.isString(str)) {
+                    return _.chain(str).words().map(function (word) {
+                        return _.capitalize(word);
+                    }).join(' ');
+                }
+                return str;
+            },
+            safeString: function (str) {
+                if (_.isString(str)) {
+                    return new Handlebars.SafeString(str);
+                }
+                return str;
+            },
+            splitDashes: function (str) {
+                return str.split('-').join(' ');
+            },
+            encodeString: function (str) {
+                if (_.isString(str)) {
+                    return encodeURIComponent(str);
+                }
+                return str;
+            },
+            debug : function() {
+            console.log("Current Context");
```

Comment:
This is for debugging only. If the debug function isn't called in a handlebars file, the code is never executed.